pkg/preparer/orchestrate.go

```diff
@@ -108,6 +115,15 @@ func (p *Preparer) WatchForPodManifestsForNode(quitAndAck chan struct{}) {
 	}
 }
 
+func (p *Preparer) podIdForbidden(id string) bool {
+	for _, forbidden := range p.forbiddenPodIds {
```

Comment:
It occurs to me that instead of doing a linear search through array, should probably look for its presence in a map instead.